### ISSUE TITLE
Feature Request: Set the IC and Manager on the Trello card automatically

### ISSUE THREAD

**Issue by reidmix** [heroku/retrodot#187]
I'd love if both the IC and Manager for the Primary Team was set on the trello card automatically.

**Comment by joshuatobin** [comment]
Could we have this set in the DB vs the card, since the plan is eventually to do away with trellos?

**Comment by reidmix** [comment]
Yeah, I suppose that will work.  Right now I use it to figure out who I should be asking questions within the trello card.  (It'll be nice to have rid of the trellos).

**Comment by lexelby** [comment]
Note: it's not immediately clear to me how Retrodot can figure out who the IC is.  Perhaps we'll have to add a new chatops command?

**Comment by joshuatobin** [comment]
@reidmix @lex is there any external source we can pull from without reinventing the wheel here?

**Comment by reidmix** [comment]
We could use the oncall schedule for IC and i bet it would be mostly right, we could have a chatops command to set it/correct it when its not.

**Comment by lexelby** [comment]
Storing the IC in retrodot has another advantage: it tells retrodot who to nag to do the retro/followup stuff like sending the email.
